packages/cli/package.json

```diff
@@ -1,80 +1,85 @@
-    "hasbin": "^1.2.3",
-    "mem-fs": "^1.1.3",
-    "mem-fs-editor": "^6.0.0",
-    "minimist": "^1.2.5",
-    "node-fetch": "^2.6.0",
-    "pkg-dir": "^4.2.0",
-    "pluralize": "^8.0.0",
-    "ts-node": "^8.9.0",
-    "tsconfig-paths": "^3.9.0",
-    "username": "^5.1.0",
-    "vinyl": "^2.2.0"
-  },
-  "devDependencies": {
-    "@blitzjs/server": "0.6.6",
-    "@oclif/dev-cli": "^1.22.2",
-    "@oclif/test": "^1.2.5",
-    "@prisma/cli": "2.0.0-beta.3",
-    "@types/pluralize": "^0.0.29"
-  },
-  "oclif": {
-    "commands": "./lib/src/commands",
-    "bin": "blitz",
-    "plugins": [
-      "@oclif/plugin-help",
-      "@oclif/plugin-not-found"
-    ]
-  },
-  "keywords": [
-    "blitz",
-    "cli"
-  ],
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/blitz-js/blitz"
-  },
-  "gitHead": "9c54cfaef19b63c30a7e41447541c7cc30b5c6d4"
+	"name": "@blitzjs/cli",
+	"description": "Blitz.js CLI",
+	"version": "0.6.6",
+	"license": "MIT",
+	"scripts": {
+		"b": "./bin/run",
+		"predev": "wait-on -d 500 ../server/dist/packages/server/src/index.d.ts",
+		"dev": "rimraf lib && yarn build:templates && tsc --watch --preserveWatchOutput",
+		"build": "rimraf lib && tsc && yarn build:templates",
+		"build:templates": "rimraf templates/**/node_modules && cpy --dot --parents '!/node_modules/' templates lib",
+		"test": "jest",
+		"test:watch": "jest --watch"
+	},
+	"author": {
+		"name": "Brandon Bayer",
+		"email": "b@bayer.ws",
+		"url": "https://twitter.com/flybayer"
+	},
+	"main": "lib/src/index.js",
+	"bin": {
+		"blitz": "./bin/run"
+	},
+	"files": [
+		"/bin",
+		"/lib"
+	],
+	"dependencies": {
+		"@oclif/command": "^1.5.20",
+		"@oclif/config": "^1.15.1",
+		"@oclif/plugin-help": "^2.2.3",
+		"@oclif/plugin-not-found": "^1.2.3",
+		"camelcase": "^6.0.0",
+		"chalk": "^4.0.0",
+		"chokidar": "^3.3.1",
+		"cross-spawn": "^7.0.2",
+		"diff": "^4.0.2",
+		"dotenv": "^8.2.0",
+		"enquirer": "^2.3.4",
+		"fs-extra": "^9.0.0",
+		"fs-readdir-recursive": "^1.1.0",
+		"globby": "^11.0.0",
+		"has-yarn": "^2.1.0",
+		"hasbin": "^1.2.3",
+		"mem-fs": "^1.1.3",
+		"mem-fs-editor": "^6.0.0",
+		"minimist": "^1.2.5",
+		"mysql2": "^2.1.0",
+		"node-fetch": "^2.6.0",
+		"pg": "^8.0.3",
+		"pkg-dir": "^4.2.0",
+		"pluralize": "^8.0.0",
+		"sqlite": "^4.0.7",
+		"sqlite3": "^4.1.1",
+		"ts-node": "^8.9.0",
+		"tsconfig-paths": "^3.9.0",
+		"username": "^5.1.0",
+		"vinyl": "^2.2.0"
+	},
+	"devDependencies": {
+		"@blitzjs/server": "0.6.6",
+		"@oclif/dev-cli": "^1.22.2",
+		"@oclif/test": "^1.2.5",
+		"@prisma/cli": "2.0.0-beta.3",
+		"@types/pg": "^7.14.3",
+		"@types/pluralize": "^0.0.29"
+	},
+	"oclif": {
+		"commands": "./lib/src/commands",
+		"bin": "blitz",
+		"plugins": [
+			"@oclif/plugin-help",
+			"@oclif/plugin-not-found"
+		]
+	},
+	"keywords": [
+		"blitz",
+		"cli"
+	],
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/blitz-js/blitz"
+	},
+	"gitHead": "9c54cfaef19b63c30a7e41447541c7cc30b5c6d4"
```

Comment:
Sorry for the delay on review here!

This pkg.json is converted to tabs. Looks like this is still left over from the prettier mess up :)